graphql/resolvers.go

```diff
@@ -986,6 +986,37 @@ func (r *queryResolver) SiteBanner(ctx context.Context) (*restModel.APIBanner, e
 	return &banner, nil
 }
 
+func (r *queryResolver) Host(ctx context.Context, hostID string) (*restModel.APIHost, error) {
```

Comment:
can you put this resolver above or below the `Hosts` resolver please so that they are in proximity to one another?